wire-ios-sync-engine/Source/Synchronization/SyncAgent.swift

```diff
@@ -139,11 +139,24 @@ final class SyncAgent: NSObject, SyncAgentProtocol {
     }
 
     private func suspend() async {
-        WireLogger.sync.debug("suspending sync")
+        let backgroundActivity = BackgroundActivityFactory.shared.startBackgroundActivity(
+            name: "suspending sync"
+        )
+
+        WireLogger.sync.debug(
+            "suspending sync \(backgroundActivity != nil ? "in a background task" : "")"
```

Comment:
todo: add sync atttributes
```suggestion
            "suspending sync \(backgroundActivity != nil ? "in a background task" : "")"
```

Reply:
done https://github.com/wireapp/wire-ios/pull/3173/commits/720fdb56505f79b80a56cb3278b3bb584e1777ce